library/std/src/thread/scoped.rs

```diff
@@ -33,7 +33,7 @@ pub struct Scope<'scope, 'env: 'scope> {
 ///
 /// See [`Scope::spawn`] for details.
 #[stable(feature = "scoped_threads", since = "1.63.0")]
-pub struct ScopedJoinHandle<'scope, T>(JoinInner<'scope, T>);
+pub struct ScopedJoinHandle<T>(JoinInner<T>);
```

Comment:
Removing a lifetime parameter from a stable item is API breaking, is it not?

Reply:
Oh, I didn't realize this is public. *oops*

Reply:
For my own confidence: `PhantomData<&'scope>` is the correct existing variance here. (The lifetime variances involved in this API is subtle and controlled by `PhantomData`, not just normal references.)